docs/data/joy/components/accordion/AccordionSizes.js

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react';
+import AccordionGroup from '@mui/joy/AccordionGroup';
+import Accordion from '@mui/joy/Accordion';
+import AccordionDetails from '@mui/joy/AccordionDetails';
+import AccordionSummary from '@mui/joy/AccordionSummary';
+import Button from '@mui/joy/Button';
+import ToggleButtonGroup from '@mui/joy/ToggleButtonGroup';
+import Stack from '@mui/joy/Stack';
+
+export default function AccordionSizes() {
+  const [size, setSize] = React.useState('md');
+  return (
+    <Stack spacing={2} sx={{ maxWidth: 400, flex: 1 }}>
+      <ToggleButtonGroup
+        size="sm"
+        buttonFlex={1}
+        value={size}
+        onChange={(event, newValue) => setSize(newValue)}
```

Comment:
There can be no value selected, expected?

<img width="446" alt="Screenshot 2023-07-27 at 15 35 58" src="https://github.com/mui/material-ui/assets/3165635/165fed37-8cfe-4bb9-bb9a-05478d6252aa">

Reply:
How to reproduce it?

Reply:
https://github.com/mui/material-ui/assets/3165635/99e6bc94-1197-457d-8c4f-acf53928807d

https://deploy-preview-38164--material-ui.netlify.app/joy-ui/react-accordion/#sizes

Oh, actually https://deploy-preview-38164--material-ui.netlify.app/joy-ui/react-toggle-button-group/#editor-toolbar has the same bug now? One of the text alignment must be active at all time.

Reply:
Hmm, I don't think so. We should let developers handle by themselve.